### Problem Statement 1: Blood glucose levels for obese patients have a mean of 100 with a standard deviation of 15. A researcher thinks that a diet high in raw cornstarch will have a positive effect on blood glucose levels. A sample of 36 patients who have tried the raw cornstarch diet have a mean glucose level of 108. Test the hypothesis that the raw cornstarch had an effect or not.

In [4]:
Sample_mean = 108
Number_of_samples = 36
Population_Mean = 100
Population_Standard_deviation = 15

import math

Null hypothesis H0: Mean = 100
Alternate hypothesis H1: Mean ≠ 100

Since there is no information provided about the significance level, we can assume it to be 0.05 or 95% confidence level. 

In [9]:
#Calculating Z score

Sample_Error = Population_Standard_deviation/math.sqrt(Number_of_samples)
Z_test_statistic = (Sample_mean - Population_Mean)/Sample_Error
Z_test_statistic

3.2

The Z value of the test statistic is 3.2. When we look up the p-value for the Z test statistic from Z table, its 0.9993. Its greater than the 0.05 significance level. Hence we can reject the null hypothesis and can conclude that the raw cornstarch had an effect on blood glucose level. 

### Problem Statement 2: In one state, 52% of the voters are Republicans, and 48% are Democrats. In a second state, 47% of the voters are Republicans, and 53% are Democrats. Suppose a simple random sample of 100 voters are surveyed from each state. What is the probability that the survey will show a greater percentage of Republican voters in the second state than in the first state?

P1 = the proportion of Republican voters in the first state
P2 = the proportion of Republican voters in the second state

p1 = the proportion of Republican voters in the sample from the first state
p2 = the proportion of Republican voters in the sample from the second state

Number of voters sampled from the first state, n1 = 100
Number of voters sampled from the second state, n2 = 100

In [12]:
n1 = 100 * 0.52 
n2 = 100 * 0.48

P1 = 0.52
P2 = 0.47
##Q1 = Proportion of non-republican voters in first place 
#Q1 = 1 - P1 
##Q2 = Proportion of non-republican voters in second state
#Q2 = 1 - P2


0.48
0.53


In [18]:
#Mean of the difference in sample proportions or the expected value E(p1 - p2)
md = P1 - P2 
md

0.050000000000000044

In [24]:
#Standard deviation of the difference 
std1 = (P1*(1-P1)/n1) 
std2 = (P2*(1-P2)/n2)
std = math.sqrt(std1+std2)


We need to find the probability that p1 is less than p2, i.e, p1 - p2 is less than 0. To find this probability, we need to transform the random variable p1 - p2 into a z-score. 

In [25]:
x = 0
Z = (x - md)/std


From the table we find that the probability of a Z-score being -0.7082 or less is 0.24. Therefore, the probability that the survey will show a greater percentage of Republican voters in the second state than in the first state is 0.24.


### Problem Statement 3: You take the SAT and score 1100. The mean score for the SAT is 1026 and the standard deviation is 209. How well did you score on the test compared to the average test taker?



In [26]:
score = 1100
mean_score = 1026
std_dev = 209

Z = (score - mean_score)/std_dev
print("Z score: ", Z)


Z score:  0.35406698564593303


The above calculation shows that the score is 0.35 standard deviations above the mean.